packages/functions/src/dal/migrationScripts/ChangeAppointmentStatusMigration.ts

```diff
@@ -1,39 +1,39 @@
-import { getAllLatestAppointments, setAppointment } from "../AppointmentDataAccessLayer";
+import {
+  getAllAppointments,
+  setAppointment,
+} from "../AppointmentDataAccessLayer";
 import { AppointmentStatus } from "@zm-blood-components/common/src";
-const fs = require('fs');
+import * as functions from "firebase-functions";
+const fs = require("fs");
 
-export async function addStatusForAppointments(){
-  const appointments = await getAllLatestAppointments();
+export async function addStatusForAppointments() {
+  const appointments = await getAllAppointments();
 
   return appointments.map((appointment) => {
     let newStatus = AppointmentStatus.AVAILABLE;
     const now = new Date();
-    if(appointment.donorId !== "")
-    {
+    if (appointment.donorId !== "") {
       if (appointment.donationStartTime.toDate() < now)
-        newStatus = AppointmentStatus.COMPLETED; // should this be completed?
-      else
-        newStatus = AppointmentStatus.BOOKED;
-    }
-    else
-    {
-      if(appointment.donationStartTime.toDate() < now)
-        newStatus = AppointmentStatus.NOSHOW
+        newStatus = AppointmentStatus.COMPLETED;
+      // should this be completed?
```

Comment:
That depends on when would we run this job.
Currently there's no way to mark an appointment as COMPLETED, so I think we should mark all appointments with a donor id with one status, probably `Booked`.

Once we roll out to production a way to confirm a donation we can go back to all past booked appointments and change them to COMPLETED.

What do you think?

Reply:
I think its a little duplicate work....how does it hurt us that these are COPMLETED now? this would just add another migration, although it has no meaning now...
BUT, I'll change it so we can roll